modules/ROOT/pages/indexes/search-performance-indexes/managing-indexes.adoc

```diff
@@ -1049,7 +1049,7 @@ The following statement will attempt to drop the index named `node_range_index_n
 [source,javascript, indent=0]
 ----
 {
-  "name": "range_index_param"
+  "name": "node_range_index_name"
```

Comment:
Not that it really matters but I think that when I added the example initially the idea was to remove the parameter created one `range_index_param` 🤷